vspace/src/main/java/edu/asu/diging/vspace/web/staff/StaffSpacesSearchController.java

```diff
@@ -0,0 +1,47 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import edu.asu.diging.vspace.core.model.ISpace;
+import edu.asu.diging.vspace.core.services.ISpaceManager;
+
+
+@Controller
+public class StaffSpacesSearchController {
+
+    @Autowired
+    private ISpaceManager spaceManager;
+
+    @RequestMapping("/staff/spaces/search")
+    public ResponseEntity < String > search(@RequestParam(value = "term", required = false) String search) {
```

Comment:
there should be no blanks between `<` and `String` and `>` (everywhere here). please follow our code style

Reply:
?